raiden/api/python.py

```diff
@@ -653,6 +658,7 @@ def get_channel_events(
                 new_event.update(event.__dict__)
                 returned_events.append(new_event)
 
+        returned_events.sort(key=lambda evt: evt.get('block_number'))
```

Comment:
Nice. But for consistency please make sure that all events endpoints follow this behaviour.